crates/credentials/src/vc.rs

```diff
@@ -36,6 +95,15 @@ pub enum Issuer {
     Object(NamedIssuer),
 }
 
+impl Issuer {
+    pub fn id(&self) -> &str {
+        match self {
+            Issuer::String(s) => s,
+            Issuer::Object(obj) => &obj.id,
+        }
+    }
+}
```

Comment:
you can actually already do this by calling `.to_string()` on an instance of `issuer`, we do this in a few places already for example go look inside the `fn create_vc(issuer: Issuer) -> VerifiableCredential` function inside the tests module

Reply:
the `impl Display for Issuer` implementation of the `fmt` method implements this functionality

Reply:
ok awesome, thanks for this lol I was pulling my hair out